style.css

```diff
@@ -0,0 +1,11 @@
+.basis {
+  font-family: "Roboto", serif;
+  height: 2766px;
+  width: 1440px;
+}
+
+/*.header {
+  background-image: url(images/nav/pic-bgr.jpg);
+  height: 839px;
+  width: 1440px;
+}*/
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6245874)